docs/pull_request_template.md

```diff
@@ -1,23 +1,33 @@
-Please note that acceptance of community PRs will be delayed while we are
-bringing our test and CI systems online. For more information, see the
-README.md.
-
 # Description
 
-# Checklist:
 
-- [ ] I understand README.md.
-- [ ] If this is a feature addition, that feature has been voted into the C++
-  Working Draft.
-- [ ] Identifiers in any product code changes are properly `_Ugly` as per
+
+# Checklist
+
+If you're unsure about a box, leave it unchecked. A maintainer will help you.
+
+If a box isn't applicable, add an explanation in **bold**.
+For example: **(N/A: this is a bugfix, not a feature)**
+
+- [ ] I understand README.md. I also understand that acceptance of
+  community PRs will be delayed until the test and CI systems are online.
+- [ ] If this is a feature addition, that feature has been voted into the
+  C++ Working Draft.
+- [ ] Identifiers in product code changes are properly `_Ugly` as per
   https://eel.is/c++draft/lex.name#3.1 .
-- [ ] Identifiers in test code changes are *not* `_Ugly`.
+- [ ] Identifiers in test code changes are *not* `_Ugly`. **(N/A: no tests)**
 - [ ] Test code includes the correct headers as per the Standard, not just
-  what happens to compile.
-- [ ] The STL builds and test harnesses have passed (must be manually verified
-  by an STL maintainer before CI is online, leave this unchecked for initial
-  submission).
-- [ ] This change introduces no known ABI breaks (adding members, renaming
-  members, adding virtual functions, changing whether a type is an aggregate or
-  trivially copyable, etc.). If unsure, leave this box unchecked and ask a
-  maintainer for help.
+  what happens to compile. **(N/A: no tests)**
+- [ ] The STL builds successfully and all tests have passed (must be manually
+  verified by an STL maintainer before CI is online, leave this unchecked for
+  initial submission).
+- [ ] These changes introduce no known ABI breaks (adding members, renaming
+  members, adding virtual functions, changing whether a type is an aggregate
+  or trivially copyable, etc.).
+- [ ] These changes were written from scratch using only this repository and
+  the C++ Working Draft as a reference (and any other cited standards).
+  If they were derived from a project that's already listed in NOTICE.txt,
+  that's fine, but please mention it. If they were derived from any other open
+  source project (including Boost and libc++, which are not yet listed in

```

Comment:
Maybe strike "open source" here? I think we'd *especially* like to know if the contribution is derived from a closed-source project.